__tests__/addressBlock.test.ts

```diff
@@ -0,0 +1,20 @@
+import { NextResponse } from "next/server";
+import { checkOFAC } from "../app/blocked/api/util";
+import { expect, test } from "@jest/globals";
+
+describe("Address blocks are in place", () => {
+  test("Address valid.", async () => {
```

Comment:
We would need to test the component as the check needs to happen on the client side.

Reply:
Tested with the local setup with some mock data manually and seems work as expected, meanwhile also add more unit tests for the lib.  So could we improve it later with another PR?

Reply:
I skipped these tests as they are testing an HTTP call to the Chainalysis service and not the fact that the UI enforces the results of the call. Created a separate ticket for this work as it will require some refactor of components: 

https://linear.app/snowfork/issue/SNO-1083